recipes/pyeda/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "pyeda" %}
+{% set version = "0.29.0" %}
+
+package:
+  name: {{ name | lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: efd4d3e548079ca059f44f9babb9bd67b332ec1762897324fcceb8da1d0f5fed
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  script_env:
+    - CFLAGS=-Wno-incompatible-pointer-types
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ stdlib("c") }}
+    - clang
```

Comment:
Why do you need `clang` here?

Reply:
AFAIK it's required for compiling PicoSAT and Espresso. I'll try without.